codeql-custom-queries-cpp/qlpack.lock.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Is this equivalent to not having a lock file at all?

Reply:
Actually, yes it is. I will remove them.

Reply:
At some point, if we want to remove the ql submodule, we will need to recreate the lock file with specific versions in it.